### ISSUE TITLE
Зависает с Pixelmon\Hangs with Pixelmon

### ISSUE THREAD

**Issue by Shurik1236** [DarkholmeTenk/TardisModIssues#608]
Лог: https://pastebin.com/6fv27X89
Darkcore: http://tardismod.com:8080/job/Darkcore/257/
Tardis Mod: http://tardismod.com:8080/job/TardisMod/356/

Проблема при комбинации Pixelmon + Tardis Mod.
Захожу в tardis, и вечный "Downloading terrain"...
Как исправить?

(Online translator)
Log: https://pastebin.com/6fv27X89
Darkcore: http://tardismod.com:8080/job/Darkcore/257/
Tardis Mod: http://tardismod.com:8080/job/TardisMod/356/

The problem with the combination Pixelmon + Tardis Mod.
I go to the tardis, and eternal "Downloading terrain"...
How to fix?

**Comment by FoxPotato** [comment]
did you change any of the config options, or maybe added schematics? Or even used an addon for the tardis mod (if those exist lmao)

**Comment by Shurik1236** [comment]
config - изначальный. Схем своих нет. Аддонов тоже нет.
Без Pixelmon всё работает.
(Online translator)
config - original. Schemes its not. Addons either.
Without Pixelmon and it works.

**Comment by SvEgiiVEteR** [comment]
@Shurik1236 а почему используешь старые версии?
why use the old version?

**Comment by Shurik1236** [comment]
Новые версии выдают при запуске это: https://pastebin.com/tm6VcLUW

**Comment by SvEgiiVEteR** [comment]
@Shurik1236 замени darkcore на darkcore-0.46-281.jar там кажется проблема с новой версией. я написал исус уже.
и насчёт твоего краша у меня тоже часто.
Caused by: java.lang.NoClassDefFoundError: appeng/api/networking/IGridHost
мод просто хочется AE2. установи его и возможно ещё тумкрафт. вообщем я на новых версиях тока так делаю.и доволно долго играл. заодно и проверишь свою проблему на новых версиях

**Comment by SvEgiiVEteR** [comment]
@Shurik1236 
MOB_Pixelmon-1.7.10-3.5.1-universal-FAxV.jar
PACK_2_darkcore-0.46-281-FA1V.jar
PACK_2_tardismod-1.7.10-0.995-362-FA1V.jar
Will check everything works

**Comment by Shurik1236** [comment]
Так, с АЕ2 запустилось... Я попробовал все моды убрать чтобы посмотреть что будет... Оказывается tardis mod ещё и waila требует!
Что то разработчик перемудрил...

А баг не исправился. Только видоизменился. Я просто теперь падаю до -2 уровня и всё. Мир не прогружается.

**Comment by SvEgiiVEteR** [comment]
@Shurik1236 ну у меня стоит ещё модов 10 + к тардису НО проблемы не было. создал новый мир взял ключь и уже там
и моежт удалить конфиги что бы пересоздались..
